care/facility/api/serializers/patient.py

```diff
@@ -258,6 +258,11 @@ def validate(self, attrs):
             raise serializers.ValidationError(
                 {"non_field_errors": ["Either age or date_of_birth should be passed"]}
             )
+        if "date_of_birth" in validated:
+            if datetime.datetime.now().year-validated['date_of_birth'].year <=0:
```

Comment:
- this method is incorrect.
- use now from django.utils.timezone
- this can be a field level validation

- add validations for year_of_birth

Reply:
Ok , I'll correct them as soon as possible

Reply:
@sainak while solving this I have also figured out that for this {"abha_number": null, "date_declared_positive": null,"blood_group":"AB+","gender":1,"age":10, "disease_status": "NEGATIVE", "emergency_phone_number": "+919000000667", "is_vaccinated": "false", "number_of_doses": 0,"facility":"81092ced-8720-44cb-b4c5-3f0ad0540153", "phone_number": "+919000044344", "srf_id": "", "test_id": ""} we are encountering a server error 
![Screenshot 2024-02-29 213944](https://github.com/coronasafe/care/assets/146672184/286554b7-465e-45dd-9385-523bf21bf08c)
should I add field-level validation for_ancedental or make its value false when not found in validated_data

Reply:
- yes you can add default=False for that field
- having a quick range check in the serializer will be good

Reply:
Ok sounds good